plugins/notification-service-extension/plugin/src/FileManager.ts

```
@@ -0,0 +1,42 @@
+    })
+  }
+
+  static async copyFile(path1: string, path2: string): Promise<void> {
+    const fileContents = await FileManager.readFile(path1)
+    await FileManager.writeFile(path2, fileContents)
+  }
+
+  static dirExists(path: string): boolean {
+    return fs.existsSync(path)
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)